### PR TITLE
nvmeof-recoverer: register nvmeofstorage controller to rook-operator

### PR DESCRIPTION
**Pull request by cheolho-kang**
**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: pkg/operator/ceph/nvmeof_recoverer/nvmeofstorage/controller.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (r *ReconcileNvmeOfStorage) Reconcile(context context.Context, request reconcile.Request) (reconcile.Result, error) {
+	logger.Debugf("reconciling NvmeOfStorage. Request.Namespace: %s, Request.Name: %s", request.Namespace, request.Name)
+
```
Comment (myoungwon) [inline pkg/operator/ceph/nvmeof_recoverer/nvmeofstorage/controller.go:107]: remove the following lines

```
	if strings.Contains(request.Name, "nvmeofstorage") {
 		// TODO (cheolho.kang): Implement the reconclie logic later
 	}

 	// Return and do not requeue
 	logger.Debug("done reconciling")
```

Comment (cheolho-kang) [reply pkg/operator/ceph/nvmeof_recoverer/nvmeofstorage/controller.go:107]: Okay,
Based on your comment, I think it's not relevant to the scope of this PR. I'll remove it.

**Comment by myoungwon** [comment]
@cheolho-kang add reconciliation logic for NvmeOfStorage <-- 제목이 커밋을 포함하는거 같지 않은데..

**Comment by myoungwon** [comment]
register nvmeofstorage controller to XXX (e.g., rook)?

**Pull request merged**